ui/src/pages/frontsite/components/Series/SeriesMetadataAccordion.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import { Accordion, Divider, Icon } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import {
+  SeriesAccessUrls,
+  SeriesAllTitles,
+  SeriesInfo,
+  SeriesUrls,
+  SeriesAlternativeTitles,
+  SeriesIdentifiers,
+} from './';
+import { EmptyMessage } from '@components';
+import isEmpty from 'lodash/isEmpty';
+
+export class SeriesMetadataAccordion extends Component {
+  state = { activeIndex: 'details' };
+
+  handleClick = (e, titleProps) => {
+    const { index } = titleProps;
+    const { activeIndex } = this.state;
+    const newIndex = activeIndex === index ? '' : index;
+
+    this.setState({ activeIndex: newIndex });
+  };
+
+  render() {
+    const { metadata } = this.props;
+    const { activeIndex } = this.state;
+    return (
+      <Accordion fluid styled>
+        <Accordion.Title
+          active={activeIndex === 'details'}
+          index="details"
+          onClick={this.handleClick}
+        >
+          <Icon name="dropdown" />
+          Details
+        </Accordion.Title>
+        <Accordion.Content active={activeIndex === 'details'}>
+          <SeriesInfo metadata={metadata} />
+          <SeriesAlternativeTitles metadata={metadata} />
+        </Accordion.Content>
+
+        <Accordion.Title
+          active={activeIndex === 'identifiers'}
+          index="identifiers"
+          onClick={this.handleClick}
+        >
+          <Icon name="dropdown" />
+          Identifiers
+        </Accordion.Title>
+        <Accordion.Content active={activeIndex === 'identifiers'}>
+          <EmptyMessage
```

Comment:
IMO this should be inside the `Identifiers` component below . The component should know what to do when the Identifiers are empty